lighthouse-core/audits/consistently-interactive.js

```diff
@@ -246,7 +246,7 @@ class ConsistentlyInteractiveMetric extends Audit {
         return {
           score,
           rawValue: timeInMs,
-          displayValue: `${displayValue.toLocaleString()}ms`,
+          displayValue: `${displayValue.toLocaleString()} ms`,
```

Comment:
devtools style is with `\xa0` which is a nbsp. good because it prevents line breaks between the two. and it's a tad narrower than normal space, often.

eg
```js
/** @type {!Common.UIStringFormat} */
UI._millisFormat = new Common.UIStringFormat('%.0f\xa0ms');

/** @type {!Common.UIStringFormat} */
UI._secondsFormat = new Common.UIStringFormat('%.2f\xa0s');
```